domain/src/main/java/com/teamagam/gimelgimel/domain/base/logging/DomainLoggerFactoryHolder.java

```diff
@@ -0,0 +1,25 @@
+package com.teamagam.gimelgimel.domain.base.logging;
+
+public class DomainLoggerFactoryHolder {
+    private static DomainLoggerFactory sDomainLoggerFactory;
+
+    public static void initialize(DomainLoggerFactory domainLoggerFactory) {
+        if (isInitialized()) {
```

Comment:
I can't initialize it by itself, since it needs to get DomainLoggerFactory.
I could have used some dagger trick to inject it with the value, but I chose to inject it manually.
The exception throwing is there to assure nobody will make a mistake further on, and given the situation I think it's a reasonable decision.